server/index.js

```diff
@@ -23,7 +23,6 @@ database.once("connected", () => {
 });
 
 const errorHandler = (err, req, res, next) => {
-  console.log(!(err instanceof CustomError));
   console.log(err);
```

Comment:
Just want to double check that we still want this console.log statement

Reply:
Oh yea the second one is there for general debug purposes, I removed the first one because I accidentally left it in when I originally implemented the error handler